CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,36 @@
+# Code of Conduct {#about_code_of_conduct}
+
+The SeqAn project adheres to the [Berlin Code of Conduct](https://berlincodeofconduct.org/). From the code of conduct:
+
+> A primary goal of all the conferences and user groups that refer to this Code of Conduct is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status and religion (or lack thereof).
+
+> A supplemental goal of this Code of Conduct is to increase open [source/culture/tech] citizenship by encouraging participants to recognize and strengthen the relationships between our actions and their effects on our community.
+
+Please have a look at the full text here: https://berlincodeofconduct.org/
+
+# Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the
+project or its community. Examples of representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed representative at an online or
+offline event.
+
+In particular it applies to:
+
+  * Communication on GitHub, including issues, pull-requests and commit messages;
+  * the seqan-dev and seqan-team mailing lists;
+  * the gitter chat rooms;
+  * SeqAn-related meetings in real life and the annual developer retreat.
```

Comment:
I would remove the reference to the annual developer meeting. We should not have named concrete events that may change in the future.